lib/Puzzle.js

```diff
@@ -18,8 +18,33 @@ class Puzzle {
   hasBeenSolved() {
     this.solved = true;
   }
+
```

Comment:
I'm curious to know your thought-process on including `getPuzzleBank` as a method under `Puzzle` instead of `Game` - considering that `Game` keeps track of the puzzle bank and `Puzzle` only needs knowledge of the puzzle it currently owns based on the round that is being played.

As it's currently set up, there is the possibility that the puzzle bank would be completely different from round to round since you are creating a new instance of `Puzzle` and then calling the `getPuzzleBank` method that is randomized and owned by `Puzzle`. Unless you were planning on only calling `Game.getPuzzleBank` the one time and then creating new instances of `Puzzle` at some other place in your `Game` file - which also seems not ideal to me.

TLDR
I think the high-level steps that I would follow to break this down are:
1. Game initialize and grabs the four puzzles you need for the entire game - which stays the same for the life of the game/4 rounds
2. When the first `Round` is instantiated, a new instance of `Puzzle` is created based on the value of Round 1 in your `puzzleBank`
3. Subsequent rounds/puzzles are instantiated by `Game` in a method that is separate from your method that initializes a new Game - that way you are not creating new puzzle banks every time.

Reply:
Our intention is as you've described--get a single bank of puzzles at the beginning of the game and pull from that during the subsequent rounds. It makes more sense as a method on the game and then passing the value to the puzzle as a new instantiation is created.